public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -2069,6 +2069,7 @@ public function manage_post_types_columns_output( $column, $post_id ) {
 				if ( ! empty( $speakers_ids ) ) {
 					$speakers = get_posts( array(
 						'post_type'      => 'wcb_speaker',
+						'post_status'    => 'any',
```

Comment:
```suggestion
						'post_status'    => array( 'publish', 'draft' ),
```

For clarity I think it would be good to be explicit about which statuses we expect to be included...